zh/access-dashboard.md

```diff
@@ -0,0 +1,135 @@
+---
+title: 访问 TiDB Dashboard
+summary: 介绍如何在 Kubernetes 环境下访问 TiDB Dashboard
+category: how-to
+---
+
+# TiDB Dashboard 指南
+
+TiDB Dashboard 是 TiDB 4.0 专门用来帮助观察与诊断整个 TiDB 集群的可视化面板，你可以在 [TiDB Dashboard](https://github.com/pingcap-incubator/tidb-dashboard) 了解详情。本篇文章将介绍如何在 Kubernetes 环境下访问 TiDB Dashboard。
+
+## 快速上手
+
+> **注意：**
+>
+> 以下教程仅为演示如何快速访问 TiDB Dashboard，请勿在生产环境中直接使用以下方法。 
+
+`TiDB Dashboard` 目前在 4.0 版本中已经内嵌在了 PD 组件中，你可以通过以下的例子在 Kubernetes 环境下快速部署一个 4.0.0-rc 版本的 TiDB 集群。运行 `kubectl apply -f` 命令，将以下 yaml 文件部署到 Kubernetes 集群中。
```

Comment:
@Yisaer v4.0.0-rc is not the recommended TiDB version because some issues are found in this version. See 
![image](https://user-images.githubusercontent.com/41534398/82537857-548d8700-9b7d-11ea-9ad0-bc5be9ba0d09.png). What about using another version for your example?